google/cloud/bigquery/bigquery_read_connection.h

```diff
@@ -46,10 +46,6 @@ using BigQueryReadLimitedErrorCountRetryPolicy =
     ::google::cloud::internal::LimitedErrorCountRetryPolicy<
         bigquery_internal::BigQueryReadRetryTraits>;
 
-void BigQueryReadReadRowsStreamingUpdater(
```

Comment:
Unfortunately this library is GA.  I think we need to define a (hand-crafted) function that maintains backwards compat.  You can mark it deprecated with the macro that expands to `[[deprecated]]` (GOOGLE_CLOUD_CPP_DEPRECATED? cannot remember right now).